keanu-python/build.gradle

```diff
@@ -93,6 +93,37 @@ task preparePythonEnvironment {
     dependsOn(":codegen:runCodeGeneration")
 }
 
+task formatApply {
+    dependsOn("preparePythonEnvironment")
+    doLast {
+        exec {
+            commandLine 'pipenv', 'run', 'yapf', '--style', 'google', '--in-place', '--recursive', '--parallel', '.'
+        }
+    }
+}
+
+task formatCheck {
+    dependsOn("preparePythonEnvironment")
+    doLast { 
+        exec {
+            // Exclude files that are generated
+            def keanuVetexPath = 'keanu/vertex/'
+            if(isFamily(FAMILY_WINDOWS)) {
```

Comment:
Talked about this offline. We should add a task to make our generated code pass the linting.